client/CServerHandler.cpp

```diff
@@ -489,7 +489,7 @@ void CServerHandler::sendMessage(const std::string & txt) const
 		std::string connectedId, playerColorId;
 		readed >> connectedId;
 		readed >> playerColorId;
-		if(connectedId.length(), playerColorId.length()) // BUG https://bugs.vcmi.eu/view.php?id=3144
+		if(connectedId.length() < playerColorId.length())
```

Comment:
```suggestion
		if(connectedId.length() && playerColorId.length())
```

Reply:
@Nordsoft91 fix it https://github.com/vcmi/vcmi/pull/1125/commits/79bc82af63ff60cfb7b41a5b944535d8fd5e46c4